src/io/github/perrymant/moneymaker/Logger.java

```diff
@@ -2,6 +2,23 @@
 
 class Logger {
     void info(final String str) {
-        System.out.println(str);
+        System.out.println("|============================================================================================================|");
+        System.out.println("|  /$$      /$$   Welcome to ...                        /$$      /$$           /$$                           |\n" +
```

Comment:
Let's imagine this application is a lot bigger. In this case you'd have multiple calls to `logger.info()`, but not all of them would want to print this header. In this case I'd go for the following approach instead.

```java
logger.info(header.get());
```
This way we can call `logger.info()` without always getting the header AND we have a class that we can test (with fixtures) - limiting the amount of static calls (the calls to `System.out.println()` in the logger).